src/styles/sidebar/components/markdown-editor.scss

```diff
@@ -0,0 +1,44 @@
+$toolbar-border: 0.1em solid $grey-3;
+
+.markdown-editor__toolbar {
+  display: flex;
+  flex-direction: row;
+
+  background-color: $white;
```

Comment:
I notice `$white` here but `black` later.

Reply:
Indeed. I'll change this to just `white`.